src/components/TodoInfo/TodoInfo.tsx

```diff
@@ -1,4 +1,32 @@
-// Add the required types and props
-export const TodoInfo = () => (
-  <>TodoInfo markup</>
-);
+import classNames from 'classnames';
+import { User } from '../../types/User';
+import { UserInfo } from '../UserInfo';
+
+type TodoInfoProps = {
+  title: string;
+  completed: boolean;
+  user: User | null;
+};
+
+export const TodoInfo: React.FC<TodoInfoProps> = ({
+  title,
+  completed,
+  user,
+}) => {
+  return (
+    <>
+      <article className={classNames(
+        'TodoInfo',
+        { 'TodoInfo--completed': completed },
+      )}
+      >
+        <h2 className="TodoInfo__title">
+          {title}
+        </h2>
+        { user
+          ? <UserInfo name={user.name} email={user.email} />
+          : null }
```

Comment:
```suggestion
        </h2>
        
        {user && <UserInfo user={user} />}
```
You can just pass the user and get all the needed props of it inside UserInfo